docs/src/pages/resources.tsx

```diff
@@ -228,17 +186,18 @@ const Resources = () => {
             </Modal>
             <h2>Videos</h2>
             <Card>
-              {youtubeIds.map(([idx, ids, cardTitle]) => (
+              {youtubeIds.map(([idx, ids, title]) => (
```

Comment:
@mistercrunch @pkdotson NIT that shouldn't hold up this PR: We can remove the index from the data arrays, and let the `map` provide it intrinsically, e.g. `{youtubeIds.map(([ids, title], idx) => (`